Solutions/Microsoft Windows SQL Server Database Audit/Hunting Queries/SQL-New_UserCreated.yaml

```diff
@@ -1,6 +1,9 @@
 id: 2b96760d-5307-44f0-94bd-8cf0ec52b1fb
 name: New User created on SQL Server
 description: |
+    This query detects new user creation from SQL Server using the SQLEvent KQL Parser function.
```

Comment:
"This query detects new user creation in SQL Server using the SQLEvent KQL Parser function."